### DISCUSSION TITLE
Zoom

### DISCUSSION THREAD

**Discussion by shakibmehr** [acoular/acoular#298] [Q&A] [answered]
I want to focus with Acoular after finding the sound source and hear the sound of that point better, can Acoular zoom in on a point?

**Comment by gherold** [comment, marked as answer]
Yes, that is possible by using a [BeamformerTime](https://www.acoular.org/api_ref/generated/generated/acoular.tbeamform.BeamformerTime.html#acoular.tbeamform.BeamformerTime) object. Its _result()_ function yields actual time signals. When setting up you processing chain (see [examples](https://www.acoular.org/auto_examples/index.html) how to do that), you would use a [grid](https://www.acoular.org/api_ref/generated/acoular.grids.html#module-acoular.grids) that contains (or only consists of) the point of interest. For listening in, you can set the beamformer object as _source_ in a [WriteWAV](https://www.acoular.org/api_ref/generated/generated/acoular.tprocess.WriteWAV.html#acoular.tprocess.WriteWAV) object and run its _save()_ function. The WriteWAV _channel_ trait in this case would refer to the indices of the grid points you used in the setup. You could also export a multi-channel wav file by using a list of several grid indices here.

**Reply by shakibmehr** [reply]
I would be grateful if you could prepare an example
